guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,44 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie for GA
+:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+:RepoRHEL7Server: rhel-7-server-satellite
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
+:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
+:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
+// :RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
```

Comment:
no equivalent, and shouldn't be required.

Reply:
@evgeni , thank you! I've removed this note.